README.md

```diff
@@ -1,3 +1,42 @@
 # Feito
 
-Automated code review in Python
+Automated code review in Python done with Prospector
+
+## Setting up
+
+### 1) Requirements
+
+A [Github OAuth Token](https://help.github.com/articles/git-automation-with-oauth-tokens/)
```

Comment:
We should describe which scopes we have to enable on GitHub settings page :hamburger:

![screenshot from 2018-02-22 13-41-49](https://user-images.githubusercontent.com/1095412/36551365-2f3cfc7e-17d6-11e8-94ab-37de0a9070d3.png)